### ISSUE TITLE
bag id not found.

### ISSUE THREAD

**Issue by Siusha** [arkayenro/arkinventory#1137]
![errorbag](https://user-images.githubusercontent.com/47328929/52234664-78fb1c80-28c2-11e9-8ca2-805b897ef805.jpg)

After the latest update, I got this today as an error. reinstalled tried rollbacks but its not working.
 Found and tried your solution  /run ArkInventory.Output({ArkInventory.ObjectInfoArray( "item:114821" )})
 the number is the item id. didn't work...   Changed several bags it doesn't help. Plzz help me to get this nice addon working again..  Don't mind to set all settings back again since they lost now.  After using this addon for so long years I don't like the idea of losing it.  Thanks, sofar for al good work on it. You will find a solution I know/hope.   Greets   Kibbeltje/Kinkylola and more :-)

**Comment by arkayenro** [comment]
somehow you have two extra bag slots.  the last bank bag slot is 11 so im not sure where 12 and 13 are coming from

is it only at load or does it happen all the time while youre playing?

if youre not getting an actual error then it should just keep working apart from the annoying output

**Comment by arkayenro** [comment]
if you feel like checking where its coming from you can edit arkinventory.lua and make the following change

near line 1321
`ArkInventory.OutputError( "unknown blizzard bag id - ", blizzard_id )`
`--error( "code failure" )`

remove the `--` from the front of the second line, that will force an error and generate a stack trace so i can tell where in the code it originated from

**Comment by Siusha** [comment]
Hello.

thanks for reaction.
Got something weird news it seems to be a serverside error  Not your addon error>
Why? I tested it, used your change and yes it gives allot txt about bags, not oke.
Having enough time I played around with the Lua. old or new don't change a thing. but to my surprise i found it is only on Alonsus it shows that error. none of the linked Anachronos and kul thiras. even so on other servers not linked on them, I didn't get the error. My 5 cents?  Keep up your good work. Thanks for your time.  Hug and Cya :-)       Kibbeltje/kinkylola  and others. 
note:  tested it on several full and near empty bag/banks& high and low lvl toons.

**Comment by Siusha** [comment]
Hello 
Sorry  for a late reaction now
Somehow the error is gone. the server has solved it?  don't know,
i found if I opened the first time it on toons on that server. 
I dropped all add-ons on that toon as possible.
turn them on again and voila error is gone. sofar dint sees it again.
thanks for a wonderful addon.  a Happy fan.

(used a backup for the rules worked well)
Greets  Kibbeltje/Kinkylola and more
